anomalous-tx-value/src/agent.config.ts

```diff
@@ -0,0 +1,47 @@
+import { Network } from "forta-agent";
+import { AgentConfig } from "./utils";
+
+const DECIMALS = 10 ** 18;
+
+// Threshold amounts for individual network tokens
+const thresholds = {
+  MAINNET_ETH: 100,
+  BNB: 40,
+  MATIC: 150000,
+  ARBITRUM_ETH: 100,
+  FTM: 600000,
+  AVAX: 10000,
+  OPTIMISM_ETH: 100

```

Comment:
the default value was set to 100 ethers so I tried to write the similar values for the other chain tokens

Reply:
I see. In that case, BNB is pretty low. You probably meant to put `400` instead of `40`. Though I think `450` BNB would be closer to the value of the rest.

Reply:
right, good catch! must be a typo I meant to put 400. 450 is nicer 👍  thanks

Reply:
fixed